src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.iOS.cs

```diff
@@ -23,7 +23,7 @@ private protected override bool TryOpenSourceSync([NotNullWhen(true)] out UIImag
 			return image != null;
 		}
 
-		private byte[] RenderAsPng(UIElement element, Size? scaledSize = null)
+		private static int RendeRenderAsBgra8_Premul(UIElement element, ref byte[]? buffer, Size? scaledSize = null)
```

Comment:
I have no Apple devices. Can anyone fix this method to return a Bgra8 buffer with premultiplication?

Reply:
Why do you need to change this part ?

Reply:
Yes, a developer which comes from UWP expects RenderTargetBitmap to return a Bgra8 buffer with alpha channel pre-multiplication and not a png buffer.